packages/ui/src/components/createInitiativeForm.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @author Sandhya
+ * @name Login
+ * @desc This component renders the login page
+ */
+
+import React from 'react';
+import { Card } from 'material-ui/Card';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import DatePicker from 'material-ui/DatePicker';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+
+const CreateInitiativeForm = ({
+  onSubmit,
+  onChange,
+  errors,
+  user,
+  pastDateCheck,
+  currentDate= new Date(),
+  totalParticipants,
+  onParticipantDropDownChange
+
+}) => (
+  <Card className="container create-initiative-page">
+  <form onSubmit={onSubmit}>
```

Comment:
Nothing happens onSubmit, it just consoles the username and password. Instead you should make an API call, if present, else store the form data in redux.